scanner_config:


    joint_publisher_topic: /panda_simulator/motion_controller/arm/joint_commands
    robot_subscriber_topic: /panda_simulator/custom_franka_state_controller/robot_state
    joint_subscriber_topic: /panda_simulator/custom_franka_state_controller/joint_states


    neutral_pose: [-0.017792060227770554, -0.7601235411041661, 0.019782607023391807, -2.342050140544315, 0.029840531355804868, 1.5411935298621688, 0.7534486589746342]
    joint_names: ['panda_joint1','panda_joint2','panda_joint3','panda_joint4','panda_joint5','panda_joint6','panda_joint7']
  
